index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main>
+        <section id="show-books-section" class="show-books-section">  
+            <h1>Awesome books</h1>
+        </section>
+        <section id="add-books-section" class="add-books-section">
+            <form action="">
+                <input type="text" class="title-input" placeholder="Title" required>
+                <input type="text" class="author-input" placeholder="Author" required>
+                <input type="submit" id="add-book-button">
+            </form>
+        </section>
+    </main>
+
+    <script src="main.js"></script>
+</body>
+</html>
```

Comment:
- I get this error when I launch your project. Please fix this issue for a better UI/UX. 🙏 
<img width="960" alt="wrwrgrg" src="https://user-images.githubusercontent.com/73607512/218574397-1995ce3e-faed-49bf-b2c4-f406e16a5e7d.PNG">